platform/ui/src/components/TableCell/TableCell.mdx

```diff
@@ -0,0 +1,36 @@
+---
+name: Table Cell
+menu: Components
+route: components/table-cell
+---
+
+import { Playground, Props, Link } from 'docz';
+import { TableCell } from '@ohif/ui';
+
+# Table Cell
+
+## Import
+
+```javascript
```

Comment:
If these doc pages don't make sense in isolation, do this instead:

- Create a new "Molecules" section in the navigation bar
- Show how all components work together to create a table

No reason to have unhelpful doc pages. Who uses `TableCell` in isolation?

Reply:
The idea of having this doc page was to basically display its component props. For a full example, in the `Basic usage` section, it's linking to the Table example. 

Your suggestion is to move `Table` to the `Molecules` menu and remove the `TableCell` doc page, for example? How we could demonstrate its props in this case?

Reply:
I agree that it should be documented, but if it can't be used in isolation, it may not be worth giving it it's own page. You could show how they're _all_ used together, and then show props tables for each component one after the other on the same MDX page that has the playground.

So they'd all be on the molecule's demo, even though it has smaller components, because those smaller component's can't be used outside of this very specific scenario/usage.

Reply:
Fixed! All props are placed together in the Table page.